src/d3/ForceGraph.js

```diff
@@ -58,12 +59,29 @@ const ForceGraph = (el, data) => {
         .attr("stroke-width", 0.1)
       .selectAll("circle")
       .data(nodes)
-      .join("circle")
-        .attr("r", 6)
-        .attr("fill", "#555")
-        .attr("cx", d => d.x)
-        .attr("cy", d => d.y)
+      .join("g")
+    
+    node
+        .attr("transform", d => `translate(${d.x}, ${d.y})`)
         .call(drag(simulation))
+        .append("circle")
+            .attr("r", 6)
```

Comment:
I think it'd be valuable to extract these common values (radius, stroke, colors, etc) to shared variables, and consolidate them in one place, so it's easier to make modifications across the application in the future.

Reply:
good idea.  got some changes planned for the whole thing so will do it there